lightgbm_ray/tests/release/tune_placement.py

```diff
@@ -0,0 +1,238 @@
+"""
+Test Ray Tune trial placement across cluster nodes.
+
+Example: Run this script on a cluster with 4 workers nodes a 4 CPUs.
+
+    ray up -y tune_cluster.yaml
+
+    ray attach tune_cluster.yaml
+
+    python /release_tests/tune_placement.py 4 4 10 10 --fake-data
+
+This starts 4 trials à 4 actors training 10 boost rounds on 10 data
+partitions per actor. This will use fake data created before training.
+
+This test will then confirm that actors of the same trial are PACKed
+on the same nodes. In practice we check that each node IP address only
+hosts actors of the same Ray Tune trial.
+"""
+
+import json
+import os
+
+import argparse
+import shutil
+import time
+from collections import defaultdict
+
+from xgboost_ray.compat import TrainingCallback
+
+import ray
+
+from ray import tune
+from ray.tune.session import get_trial_id
+from ray.tune.integration.docker import DockerSyncer
+from ray.util import get_node_ip_address
+
+from benchmark_cpu_gpu import train_ray
+from xgboost_ray import RayParams
+from xgboost_ray.session import put_queue
+from xgboost_ray.tests.utils import create_parquet
+from xgboost_ray.tune import TuneReportCallback
```

Comment:
Is this intentionally testing `xgboost_ray`?

Reply:
This whole PR is unnecessary!